src/content/docs/serverless-function-monitoring/aws-lambda-monitoring/opentelemetry/lambda-opentelemetry-dotnet.mdx

```diff
@@ -24,36 +24,11 @@ This guide assumes you have the following:
 * An AWS account. If you don't have one, [create one for free](https://aws.amazon.com/).
 * A .NET Lambda Function. If you don't have one yet, [create one now](https://docs.aws.amazon.com/lambda/latest/dg/lambda-csharp.html).
 
-## Step 1: Enable X-Ray [#xray]
-
-Even though we're not using AWS X-Ray in this setup, we still need to enable X-Ray in the Lambda function to enable the tracing features built into AWS OpenDistro.
-
-To enable X-Ray:
-
-1. Open your function in the [Lambda Console](https://console.aws.amazon.com/lambda).
-2. Choose **Configuration**, then **Monitoring Tools**.
-3. Choose **Edit**.
-4. Under **X-Ray**, enable **Active tracing**.
-
-Alternatively, if you're using [SAM (Serverless Application Model)](https://aws.amazon.com/serverless/sam/) or [CloudFormation](https://aws.amazon.com/cloudformation/resources/templates/) templates, you can enable X-Ray by adding the following to your Lambda function's properties:
-
-```yaml
-  yourFunctionHere:
-    Type: AWS::Serverless::Function
-    Properties:
-      ...
-      Policies:
-        - AWSLambdaBasicExecutionRole
-        ...
-        - AWSXrayWriteOnlyAccess
-      Tracing: Active
-```
-
 <Callout variant="important">
-  This option requires that you have `AWSLambdaBasicExecutionRole` or an equivalent policy attached to your function.
+  Enabling X-Ray is no longer needed, because `DisableAwsXRayContextExtraction` is set to `true` in the AWS OTel .NET SDK for Lambda. More details are described in [AWS OTel .NET SDK for Lambda Readme](https://github.com/open-telemetry/opentelemetry-dotnet-contrib/blob/main/src/OpenTelemetry.Instrumentation.AWSLambda/README.md#instrumentation).
```

Comment:
"You can find more details in the [AWS OTel...", again to remove the passive voice.